content/influxdb/v2.0/visualize-data/variables/variable-types.md

```diff
@@ -24,14 +24,25 @@ values are used when actually processing the query.
 The most common use case for map variables is aliasing simple, human-readable keys
 to complex values.
 
-##### Map variable CSV example
+##### Map variable example
 ```js
 Juanito MacNeil,"5TKl6l8i4idg15Fxxe4P"
 Astrophel Chaudhary,"bDhZbuVj5RV94NcFXZPm"
 Ochieng Benes,"YIhg6SoMKRUH8FMlHs3V"
 Mila Emile,"o61AhpOGr5aO3cYVArC0"
 ```
 
+With `map_variable_example` defined as above,
+choose the desired value for the variable in the script editor when entering a query like this:
+
+```js
+from(bucket: "telegraf")
+  |> range(start: v.timeRangeStart, stop: v.timeRangeStop)
+  |> filter(fn: (r) => r["_measurement] == "cpu" )
+  |> filter(fn: (r) => r["_field"] == "usage_user" )
+  |> filter(fn: (r) => r["cpu"] == v.map_variable_example)
+```
+
```

Comment:
I wouldn't nest this under "Map" because the usage isn't exclusive to Map variables. We need a new section on this doc that walks through using dashboard variables. Map, query, and CSV variables simply build the list of possible variable values in different ways, but using custom variables in a query is the same for all of them.